utils/src/tree/isAzExtParentTreeItem.ts

```diff
@@ -0,0 +1,19 @@
+/*---------------------------------------------------------------------------------------------
+*  Copyright (c) Microsoft Corporation. All rights reserved.
+*  Licensed under the MIT License. See License.txt in the project root for license information.
+*--------------------------------------------------------------------------------------------*/
+
+import type { AzExtParentTreeItem } from "./AzExtParentTreeItem";
+import type { AzExtTreeItem } from "./AzExtTreeItem";
+import type { IAzExtParentTreeItemInternal } from "./InternalInterfaces";
+
+/**
+ * Using instanceof AzExtParentTreeItem causes issues since each extension has their own version of the utils. Instead, check _isAzExtParentTreeItem
+ */
+export function isAzExtParentTreeItem(maybeParentTreeItem: unknown): maybeParentTreeItem is AzExtParentTreeItem {
+    return typeof maybeParentTreeItem === 'object' && (maybeParentTreeItem as IAzExtParentTreeItemInternal)._isAzExtParentTreeItem === true;
```

Comment:
An alternative that's about the same length:
```suggestion
    return isAzExtTreeItem(maybeParentTreeItem) && (maybeParentTreeItem as IAzExtParentTreeItemInternal)._isAzExtParentTreeItem === true;
```
Not critical one way or the other but I like taking advantage of the other `isBlah` methods when dealing with inherited classes.